.evergreen/config.in.yml

```diff
@@ -1238,10 +1260,12 @@ task_groups:
   - name: test_atlas_task_group
     setup_group:
       - func: fetch source
+      - func: assume secrets manager role
       - command: subprocess.exec
         params:
           working_dir: src
           binary: bash
+          add_expansions_to_env: true
```

Comment:
I assume this wouldn't set any of the env variables below, but double checking just in case